app/src/components/market/steps/items/funding_and_fee_step.tsx

```diff
@@ -96,22 +94,11 @@ const FundingAndFeeStep = (props: Props) => {
           description: `The fee taken from every trade. Temporarily fixed at ${MARKET_FEE}%.`,
         }}
       />
-      <FormRow
-        formField={
-          <CustomizableTokensSelect
-            addCustomValue={addCollateralCustom}
-            context={context}
-            customValues={collateralsCustom}
-            name="collateralId"
-            onCollateralChange={handleCollateralChange}
-            value={collateral}
-          />
-        }
-        title={'Collateral token'}
-        tooltip={{
-          id: `collateralToken`,
-          description: `Select the token you want to use as collateral.`,
-        }}
+      <Tokens
+        name="collateralId"
+        networkId={context.networkId}
+        onTokenChange={handleCollateralChange}
+        value={collateral}
```

Comment:
Mmmm  I think we need to wrapp this in a `FormRow` like this

```
    <FormRow
        formField={
            <Tokens
                name="collateralId"
                networkId={context.networkId}
                onTokenChange={handleCollateralChange}
                value={collateral}
            />
        }
        title={'Collateral token'}
        tooltip={{
            id: `collateralToken`,
            description: `Select the token you want to use as collateral.`,
        }}
    />
```

Reply:
Done!